enterprise_access/apps/bffs/serializers.py

```diff
@@ -0,0 +1,40 @@
+"""
+Serializers for bffs.
+"""
+from rest_framework import serializers
+
+
+class ErrorSerializer(serializers.Serializer):
+    """
+    Serializer for error.
+
+    Fields:
+        user_message (str): A user-friendly error message.
+        developer_message (str): A more detailed error message for debugging purposes.
+    """
+    developer_message = serializers.CharField()
+    user_message = serializers.CharField()
+
+    def create(self, validated_data):
+        return validated_data
+
+    def update(self, instance, validated_data):
+        return validated_data
```

Comment:
Included to fix linting error of _abstract-method_
![Screenshot 2024-10-31 at 10 34 38 AM](https://github.com/user-attachments/assets/8d0b6d00-db39-462c-a048-54374edea2c5)